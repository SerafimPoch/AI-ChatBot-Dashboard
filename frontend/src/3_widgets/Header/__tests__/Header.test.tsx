import React from "react";
import { render } from "@testing-library/react";
import {
  HeaderContainer,
  HeaderTitle,
  SwitcherWrapper,
  HeaderNavbar,
} from "../Header.styles";

describe("Header page components", () => {
  it("HeaderContainer renders correctly", () => {
    const { asFragment } = render(<HeaderContainer />);
    expect(asFragment()).toMatchSnapshot();
  });
  it("HeaderTitle renders correctly", () => {
    const { asFragment } = render(<HeaderTitle />);
    expect(asFragment()).toMatchSnapshot();
  });
  it("SwitcherWrapper renders correctly", () => {
    const { asFragment } = render(<SwitcherWrapper />);
    expect(asFragment()).toMatchSnapshot();
  });
  it("HeaderNavbar renders correctly", () => {
    const { asFragment } = render(<HeaderNavbar />);
    expect(asFragment()).toMatchSnapshot();
  });
});
