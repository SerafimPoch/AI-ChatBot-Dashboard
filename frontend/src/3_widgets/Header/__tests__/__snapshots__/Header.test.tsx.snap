// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header page components HeaderContainer renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  justify-content: space-between;
  margin-top: 24px;
  margin-bottom: 68px;
}

@media (max-width: 1550px) {
  .emotion-0 {
    margin-bottom: 116px;
  }
}

@media (max-width: 1000px) {
  .emotion-0 {
    margin-bottom: 50px;
  }
}

<div
    class="emotion-0"
  />
</DocumentFragment>
`;

exports[`Header page components HeaderNavbar renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  gap: 32px;
}

<div
    class="emotion-0"
  />
</DocumentFragment>
`;

exports[`Header page components HeaderTitle renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  font-size: 36px;
  font-weight: 700;
  line-height: 32px;
  color: undefined;
  margin-right: 40px;
  font-family: var(--font-sf-pro-display);
}

<h3
    class="emotion-0"
  />
</DocumentFragment>
`;

exports[`Header page components SwitcherWrapper renders correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
    class="emotion-0"
  />
</DocumentFragment>
`;
